bazel.sh

```diff
@@ -19,21 +19,22 @@
 # This script will clean, build, and run tests against each installation of Xcode available on the
 # machine using bazel.
 #
-# Arguments:
-#   1. bazel action (build or test, usually)
-#   2. BUILD target.
-#   3. Minimum Xcode version. E.g. "8" or "8.2.1"
+# Ordered arguments:
+#   1. action:  bazel ACTION. E.g. "build" or "test".
+#   2. target:  Bazel build target. E.g. "//path/to/target:Target"
+#
+# Named arguments:
+#   -m|--min-xcode-version <version>: Minimum Xcode version. E.g. "8" or "8.2.1"
```

Comment:
If I have three Xcodes installed:
* Xcode 8.2.0
* Xcode 8.3.3
* Xcode 9.0

And I specify a minimum version of "8.3.3", will it execute with either 8.3.3 or 9.0? With both?

Reply:
It'll execute every version of Xcode greater than or equal to the minimum xcode version.

Reply:
Added clarification to the comment.